packages/eslint-plugin-pf-codemods/lib/rules/wizard-rename-hasBodyPadding.js

```diff
@@ -1,10 +1,42 @@
-const { renameProp } = require('../helpers');
+const { getPackageImports } = require('../helpers');
 
 // https://github.com/patternfly/patternfly-react/pull/4136
 module.exports = {
-  create: renameProp(
-    ['Wizard', 'WizardBody', 'WizardToggle'],
-    {'hasBodyPadding': 'hasNoBodyPadding'},
-    node => `hasBodyPadding prop has been removed for ${node.name.name}. Use hasNoBodyPadding instead`
-  )
-};
\ No newline at end of file
+  create: function(context) {
+    const imports = getPackageImports(context, '@patternfly/react-core')
+      .filter(specifier => specifier.imported.name === 'Wizard');
+    const sourceCode = context.getSourceCode();
+      
+    return imports.length == 0 ? {} : {
+      JSXOpeningElement(node) {
+        if (imports.map(imp => imp.local.name).includes(node.name.name)) {
+          const attribute = node.attributes.find(node => node.name && node.name.name === 'hasBodyPadding');
+          if (attribute) {
+            context.report({
+              node,
+              message: 'The Wizard hasBodyPadding prop was removed in favor of hasNoBodyPadding',
```

Comment:
Message can be changed back to include `node.name.name`: `hasBodyPadding prop has been removed for ${node.name.name}. Use hasNoBodyPadding instead`